src/pages/Home.vue

```diff
@@ -84,8 +105,13 @@ export default {
     },
   },
   async created() {
-    /** Group name is sent to the API to retrieve all details */
-    this.groupData = await groupAPIService.getGroupData(this.group);
+    /** If sessionId exists in route, then retrieve session details. Otherwise, fallback is using group. */
+    if(this.sessionId === ""){
+      this.groupData = await groupAPIService.getGroupData(this.group); }
+    else{
+     this.sessionData = await sessionAPIService.getSessionData(this.sessionId);
+      (this.sessionData.enabled == "on") ? this.sessionEnabled = true : this.sessionEnabled = false;
```

Comment:
Isn't sessionData.enabled a boolean? If it's not, shouldn't it be a boolean instead of a string?

Reply:
so actually the way it is stored from google sheets is "on" or "off". What I can do is while writing to firestore, I can store it as true or false.

Reply:
yes that would be the correct way to do it, it removes dependence on strings.
if someone types "no " with a space, then we'll run into problems.

While writing to firestore, you can trim the string, make it lowercase and check if it's "yes" or "no" and save the value as boolean